src/server/auth/server/config.go

```diff
@@ -42,6 +42,7 @@ type canonicalGitHubIDP struct{}
 
 type canonicalOIDCIDP struct {
 	Issuer              string
+	IssuerOverride      string
```

Comment:
`IssuerOverride` tells pachd to try and reach Dex at a different hostname than is specified in `Issuer`. This is necessary if DNS resolves differently within the cluster and outside of it. For example, when we're running in minikube Dex is reachable at `localhost:30658` from the host machine, but at `pachd:658` from within the pachd pod. `Issuer` should be configured so it's accurate from the host machine's perspective, so the OAuth redirect links resolve properly in the user's browser.

Reply:
This comment is super helpful (especially because it kind of explains both `Issuer` and `IssuerOverride`)—how would you feel about using it as a field comment (i.e. in the code)?